db/migrate/20150320160909_add_send_build_merge_email.rb

```diff
@@ -0,0 +1,5 @@
+class AddSendBuildMergeEmails < ActiveRecord::Migration
+  def change
+    add_column :repositories, :send_build_merge_email, :boolean, default: true, null: false
```

Comment:
re: naming, maybe change `send_build_merge_email` to `send_merge_successful_email`? Looks like the other `send_X_email` fields are patterned after the templates/mailer methods themselves.